### ISSUE TITLE
GML not rendered correctly

### ISSUE THREAD

**Issue by antoniolocandro** [qgis/QGIS#35393]
**Describe the bug**

If I add the following attached file the GML should all be over the North Atlantic between USA and Europe, however when point features use the following root element aixm:ElevatedPoint (<aixm:AeronauticalGroundLight gml:id="uuid.9481f274-f05b-4c00-9017-eae75d33c45b">) it is not rendered correctly as opposed to if used aixm:Point (<aixm:DesignatedPoint gml:id="uuid.2fea81e6-023b-4069-b8cd-2ab96da27862">)

**How to Reproduce**

Actually any version has the same issue

QGIS version
3.8.3-Zanzibar
QGIS code revision
685d8b15d2
Compiled against Qt
5.11.2
Running against Qt
5.11.2
Compiled against GDAL/OGR
2.4.1
Running against GDAL/OGR
2.4.1
Compiled against GEOS
3.7.2-CAPI-1.11.0
Running against GEOS
3.7.2-CAPI-1.11.0 b55d2125
PostgreSQL Client Version
10.8
SpatiaLite Version
4.3.0
QWT Version
6.1.3
QScintilla2 Version
2.10.8
Compiled against PROJ
5.2.0
Running against PROJ
Rel. 5.2.0, September 15th, 2018
OS Version
Windows 10 (10.0)

**Additional context**

You can see how it should look here 
[EA_AIP_DS_FULL_20170701.zip](https://github.com/qgis/QGIS/files/4390570/EA_AIP_DS_FULL_20170701.zip)
![LUCIAD_VIEW](https://user-images.githubusercontent.com/4627073/77706160-35acb300-6f87-11ea-8c68-efd265fb48de.png)

**Comment by gioman** [comment]
>  3.8.3-Zanzibar

@antoniolocandro this is too old. You must re-check on the latest 3.10/12 releases. Thanks.

**Comment by gioman** [comment]
The datasource has a ton of layers/tables, what we should use to check if it projects were expected?

**Comment by antoniolocandro** [comment]
Using latest version from OSGEO4W on WIN10

QGIS version
3.13.0-Master
QGIS code revision
2d0ffedc70
Compiled against Qt
5.11.2
Running against Qt
5.11.2
Compiled against GDAL/OGR
3.1.0dev
Running against GDAL/OGR
3.1.0dev
Compiled against GEOS
3.8.1-CAPI-1.13.3
Running against GEOS
3.8.1-CAPI-1.13.3
Compiled against SQLite
3.29.0
Running against SQLite
3.29.0
PostgreSQL Client Version
11.5
SpatiaLite Version
4.3.0
QWT Version
6.1.3
QScintilla2 Version
2.10.8
Compiled against PROJ
7.1.0
Running against PROJ
Rel. 7.1.0, August 1st, 2020
OS Version
Windows 10 (10.0)
This copy of QGIS writes debugging output.
Active python plugins
contour; 
DigitizingTools; 
MemoryLayerSaver; 
Qgis2threejs; 
quick_map_services; 
scriptrunner3; 
db_manager; 
MetaSearch; 
processing

I add the filed attached before and will select only the following layers as shown per the image
![layers_for_test](https://user-images.githubusercontent.com/4627073/77767860-4c92ea00-7007-11ea-9c54-ba7e2769767e.png)

You will notice the following view on the map canvas
![qgis_map_view](https://user-images.githubusercontent.com/4627073/77768044-8d8afe80-7007-11ea-982a-b6b8289987f9.png)

Ok so like you mention has a lot of layers but I will start with the two layers I mention above

I have concluded the following

->    render correctly (check layer = AeronauticalGroundLight )

<aixm:**Point** srsName="urn:ogc:def:crs:EPSG::4326" gml:id="pBOR10">
<gml:pos>51.529284 -33.555661</gml:pos>
<aixm:horizontalAccuracy xsi:nil="true" nilReason="unknown"/>
</aixm:Point>

->  render incorrectly (check layer = DesignatedPoint )

<aixm:**ElevatedPoint** srsName="urn:ogc:def:crs:EPSG::4326" gml:id="ID_343">
<gml:pos>55.36666666666667 -33.983333333333334</gml:pos>
</aixm:ElevatedPoint>

So why does QGIS handles improperly the second case, is like it doesn't take into consideration the axis order for it which is lat,lon for EPSG:4326, I suppose because it is a specialization class from the aviation schema, but it is supposed to be inheriting properties from the GML, here is the definition for it which mentions "This is a specialization of the Point class and it inherits all attributes and associations of that class." which in turn the Point class "This is a specialization of the GM_Point class and it inherits all attributes and associations of that class." 

All of this is supposed to be based on ISO 19107
http://www.aixm.aero/sites/aixm.aero/files/imce/AIXM511HTML/AIXM/Class_ElevatedPoint.html?menu=open 

So to sum up the layer AeronauticalGroundLight has the axis reversed and uses AIXM:ElevatedPoint, the layer DesignatedPoint has the axis correct but uses AIXM:Point

**Comment by gioman** [comment]
> So why does QGIS handles improperly the second case

@antoniolocandro have you checked if this is a GDAL/OGR issue instead?

**Comment by antoniolocandro** [comment]
If you tell me what I can do to check with GDAL/OGR I could try it, I suppose it's using the driver to read/write the features so it could actually be there the issue. I am not so good with the cmd line but can certainly try if directed

**Comment by gioman** [comment]
@antoniolocandro 

```
ogrinfo -so EA_AIP_DS_FULL_20170701.xml AeronauticalGroundLight
Had to open data source read-only.
INFO: Open of `EA_AIP_DS_FULL_20170701.xml'
      using driver `GML' successful.

Layer name: AeronauticalGroundLight
Geometry: Point
Feature Count: 2
Extent: (43.200000, -33.983330) - (55.366670, -33.366670)
Layer SRS WKT:
(unknown)
identifier: String (36.0)
beginPosition: String (20.0)
interpretation: String (8.0)
sequenceNumber: Integer (0.0)
timeSlice|AeronauticalGroundLightTimeSlice|featureLifetime|TimePeriod|beginPosition: String (20.0)
name: String (5.0)
type: String (13.0)
colour: String (15.0)
flashing: String (3.0)
note: String (120.0)
```

"AeronauticalGroundLight" has no CRS and it's extent (as seen by OGR) is exactly the one where QGIS projects the points. So I guess that if there is an issue, this is in OGR or maybe the data itself.

**Comment by antoniolocandro** [comment]
Ok perfect! I will copy your commands for future reference, the actual XML has declared EPSG:4326 for each feature, so it is something to do with the driver not understanding the aixm:ElevatedPoint tag inherits from aixm:Point which in turn goes all the way to the GML profile. I will report in GDAL thanks!

**Comment by Mick33700** [comment]
Hi,
I am new on this forum.
After loading AIXM 5 XML GML data set, on QGIS :
1. Save As the layer to a Shape file format (name field limitation lenght not fit for AIXM 5)
2. use plug in SWAP XP that works on previous version of QGIS.

Other data set are available on [https://ext.eurocontrol.int/aixm_confluence/display/AIX/Overview]

**Comment by antoniolocandro** [comment]
Thanks, yes I can probably do some swap at some point and probably would be better to do an import through custom scripting into Postgis and not loose the name fields due to limitation. I am exploring the idea of not needing this and loading directly the GML overall it works, I will try shapefile or postgis but would rather have a solution where I can just swap the GML directly as it is less work

**Comment by antoniolocandro** [comment]
Just for general knowledge fixed upstream https://github.com/OSGeo/gdal/issues/2356
